.github/ISSUE_TEMPLATE/FEATURE_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: ""
+labels: ""
+assignees: ""
+---
+
+\*\*Please visit the [Trailblazer Community](https://trailhead.salesforce.com/trailblazer-community/groups/0F94V000000oRJsSAM?tab=discussion&sort=LAST_MODIFIED_DATE_DESC)
```

Comment:
For this part, could we move that out to the `config.yml` to [match](https://github.com/forcedotcom/salesforcedx-vscode/blob/eb6d17987e9193498bda177b79f7b4c800c2c1e8/.github/ISSUE_TEMPLATE/config.yml) what we're doing in the extensions?